ftw/solr/zcml.py

```diff
@@ -28,9 +28,16 @@ class ISolrConnectionConfigDirective(Interface):
         required=True,
     )
 
+    upload_blobs = schema.Bool(
+        title=u"Upload blobs to extract handler via HTTP POST.",
```

Comment:
I would call the title (label) "Upload Blobs" only.
The description already contains the explanation.